robottelo/ui/base.py

```diff
@@ -130,6 +130,31 @@ def search(self, element_name):
             time.sleep(1)
         return None
 
+    def create_a_bookmark(self, name=None, query=None, public=None,
```

Comment:
I think that make sense. perhaps I would advise another name for the method, e.g. `create_search_bookmark()` to indicate it's search related. I think using articles (a/an/the) in the names is a waste.